platform/android/src/test/render_test_runner.cpp

```diff
@@ -41,17 +42,178 @@ void Log::platformRecord(EventSeverity severity, const std::string& msg) {
 
 } // namespace mbgl
 
+namespace {
+std::string jstring2string(JNIEnv* env, jstring jStr) {
```

Comment:
toStdString()

Reply:
@mapbox/core-sdk could you pls advice on implementation here (convert jstring -> std::string with JNI)?

Reply:
Is there any special requirement here or would `jni::Make<std::string>(env, jstr)` be enough?

Reply:
We use ndk jni.h instead of mapbx base jni.hpp, it does not support `jni::Make<std::string>(env, jstr)`, we agree to keep this helper function for now